File: bubble_sort/bubbleSort.js
function bubbleSort (nums) {
    let swapped = true;
   
    while(swapped) {
      swapped = false
    for (let i=0; i < nums.length; i++) {
        if (nums[i] > nums[i+1]){
          swapped = true;
          let first = nums[i];
          let last = nums[i+1];
          nums[i] = last;
          nums[i+1] = first;
        }
      }  
    }
    return nums;
}

File: bubble_sort/bubbleSort.py
def bubbleSort(nums):
    swapped = True
    while(swapped):
        swapped = False
        for index in range(len(nums)-1):
            first = nums[index]
            next = nums[index + 1]
            if first > next:
                swapped = True
                nums[index] = next
                nums[index + 1] = first
    return nums
